tbtadm/controller.cpp

```diff
@@ -220,10 +223,9 @@ int findSL()
 {
     if (fs::exists(sysfsDevicesPath))
     {
-        for (auto dir : boost::make_iterator_range(
-                 fs::directory_iterator(sysfsDevicesPath), {}))
+        for (auto& dir : fs::directory_iterator(sysfsDevicesPath))
         {
-            if (dir.status().type() != fs::directory_file)
+            if (!is_directory(dir))
```

Comment:
Oh, nice!
But where this is coming from? How it's compiled without `fs::`?

Reply:
This is because of Argument-dependent name lookup (Koenig lookup). 
http://www.drdobbs.com/cpp/a-personal-note-about-argument-dependent/232901443

Reply:
Oh, ADL, right! Silly me.

Reply:
We can prepend ` fs::` to make it more clear.

Reply:
No need. It's better this way, I think.